dev-python/busfactor/metadata.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>harald.weiner@jku.at</email>
+		<name>Harald Weiner</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>sci-mathematics@gentoo.org</email>
+		<name>Gentoo Mathematics Project</name>
```

Comment:
Doesn't matter, you can make it **sci** or **sci-mathematics**, either is acceptable